### DISCUSSION TITLE
项目的服务端在哪

### DISCUSSION THREAD

**Discussion by sl1011** [lobehub/lobehub#2475] [Development Question] [answered]
### 💻 Operating System

Ubuntu

### 📦 Environment

Docker

### 🌐 Browser

Chrome

### 🐛 Bug Description

这个项目的服务端在哪可以看见勒，看项目里面有调用接口地址，但是不知道这个项目的服务端在哪

[a comment by an automated account is not shown]

**Comment by sl1011** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/58358677/d6474dea-421f-4e45-94d8-027dd34b6971)
例如这个接口对应的服务端在哪？

[a comment by an automated account is not shown]

**Comment by wiirhan** [comment, marked as answer]
这是nextjs项目，接口对应的文件在src/app/api/market/route.ts中。建议你先看看nextjs的官方文档

[a comment by an automated account is not shown]
